config/checkstyle/suppressions.xml

```diff
@@ -7,4 +7,8 @@
 <suppressions>
     <suppress checks="FileLength"
               files="DefaultBeanContext.java|BeanDefinitionWriter.java|DefaultHttpClient.java"/>
+    <suppress checks="DeclarationOrder"
+              files="UriTemplate.java"/>
```

Comment:
Needed to suppress this for DeclarationOrder since one of the protected static variables relies on a private static variable. It wants public -> protected -> private variables listed in that order and i did not want to expose the private variable at protected scope to make it happy.